### PR TITLE
Add error message guidelines and how to build a PDF locally

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: doc/source/guidelines/doc_practices.rst [hunk 1]
```diff
@@ -46,9 +51,9 @@ recommend the use of numpy-style docstrings so that there is consistency
 across PyAnsys libraries. For more information, see :ref:`api_documentation`.
 
 RST Files
----------
+~~~~~~~~~
 To provide general usage information in your documentation, use your favorite editor to
-create RST files that you then place in the repository's ``doc/`` directory. The ``index.rst``
+create RST files that you then place in the repository's ``doc\`` directory. The ``index.rst``
```
Comment (akaszynski) [inline doc/source/guidelines/doc_practices.rst:56]:
```suggestion
create RST files that you then place in the repository's ``doc`` directory. The ``index.rst``
```

The `/` works both on Linux, Windows, and MacOS, but `\` will only work on Windows. We can also just remove this character.

File: doc/source/guidelines/doc_practices.rst [hunk 2]
```diff
@@ -247,13 +187,15 @@ Using Windows PowerShell to Clone a GitHub Repository
 
 Using GitBash to Clone a GitHub Repository
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To use GitBase to clone a GitHub repository:
```
Comment (akaszynski) [inline doc/source/guidelines/doc_practices.rst:190]:
```suggestion
To use GitBash to clone a GitHub repository:
```

**Review by akaszynski — APPROVED** [review]
Approved with suggestions.

File: doc/source/guidelines/doc_practices.rst [hunk 3]
```diff
@@ -378,3 +318,120 @@ the GitHub PR:
...
+#. If you have not yet cloned the ``pyansys-sphinx-theme`` repository, clone it.
+
+Installing Build Requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
+You can build documentation for the ``dev_guide`` and ``pyansys-sphinx-theme`` repositories without
+installing a PyAnsys library in development mode. However, when you push changes that you have made
+in a local branch to the GitHub repository for other PyAnsys libraries, CI checks typically require
+that the full library is installed.
+
+#. In Anaconda Powershell, navigate to the base directory in the library's cloned repository.
```
Comment (MaxJPRey) [inline doc/source/guidelines/doc_practices.rst:350]:
```suggestion
#. In Anaconda command prompt, navigate to the base directory in the library's cloned repository.
```

File: doc/source/guidelines/doc_practices.rst [hunk 4]
```diff
@@ -378,3 +318,120 @@ the GitHub PR:
...
+locally:
+
+#. Install `MiKTeX <https://miktex.org/download>`_ for Windows, selecting the
+   recommended option for installing it for only your own use.
+#. From the Windows Start menu, start the MiKTeX Console.
+#. On the Welcome page, click ``Check for updates`` and install any available
+   updates.
+#. Install the latest version of Strawbery Perl, which enables you to build
+   LaTeX files, accepting the default installation location (``C:\Strawberry\``).
+#. In the Command Prompt window, type ``perl-v`` to test that you installation is
```
Comment (MaxJPRey) [inline doc/source/guidelines/doc_practices.rst:414]:
```suggestion
#. In the Command Prompt window, type ``perl-v`` to test that your installation is
```

File: doc/source/guidelines/doc_practices.rst [hunk 5]
```diff
@@ -378,3 +318,120 @@ the GitHub PR:
...
+
+#. If a PyAnsys library already exists, create a directory in which to place a clone of its GitHub repository.
+
+#. Clone the PyAnsys library's GitHub repository. For more information, see :ref:`cloning`.
+
+#. If you have not yet cloned the ``pyansys-sphinx-theme`` repository, clone it.
+
+Installing Build Requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
+You can build documentation for the ``dev_guide`` and ``pyansys-sphinx-theme`` repositories without
```
Comment (PipKat) [inline doc/source/guidelines/doc_practices.rst]:
```suggestion
You can build documentation for the `dev_guide <https://github.com/pyansys/dev-guide>`_ and `pyansys-sphinx-theme <https://github.com/pyansys/pyansys-sphinx-theme>`_ repositories without
```

**Pull request merged**